src/palladio/NodeParameter.h

```diff
@@ -159,7 +178,13 @@ static PRM_Template PARAM_TEMPLATES[] = {
                      START_RULE_HELP.c_str()),
         PRM_Template(PRM_ORD, PRM_Template::PRM_EXPORT_MAX, 1, &CommonNodeParams::LOG_LEVEL,
                      &CommonNodeParams::DEFAULT_LOG_LEVEL, &CommonNodeParams::logLevelMenu),
-        PRM_Template()};
+        PRM_Template(PRM_INT | PRM_TYPE_JOIN_NEXT | PRM_TYPE_PLAIN, 1, &SEED, PRMzeroDefaults, nullptr, &SEED_RANGE,
+                     PRM_Callback(), nullptr,
+                     1, SEED_HELP.c_str()),
+        PRM_Template(PRM_TOGGLE, 1, &OVERRIDE_SEED, PRMzeroDefaults, nullptr, nullptr, PRM_Callback(), nullptr, 1,
+                     SEED_HELP.c_str()),
```

Comment:
Use a different help text: "Force setting the random seed even if the corresponding primitive attribute already exists."

Reply:
changed help text